### PR TITLE
Login css

### PR DESCRIPTION
**Pull request by smriteemanandhar**
I have added css in login page and modified login page with some html. Please have a look.

### CODE REVIEW

File: src/components/Login.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@700&family=Poppins:wght@400;500;600&display=swap');
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
```
Comment (sanjayguwaju) [inline src/components/Login.css]: It adds css for whole website so don't use it.

File: src/components/Login.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+body{
+  margin: 0;
+  padding: 0;
+  background: linear-gradient(120deg,#2980b9, #8e44ad);
+  height: 100vh;
+  overflow: hidden;
+}
```
Comment (sanjayguwaju) [inline src/components/Login.css]: We don't have anybody tag element in the Login.jsx component so don't use it.

File: src/components/Login.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  height: 50px;
+  border: 1px solid;
+  background: #2691d9;
+  border-radius: 25px;
+  font-size: 18px;
+  color: #e9f4fb;
+  font-weight: 700;
+  cursor: pointer;
+  outline: none;
+}
```
Comment (sanjayguwaju) [inline src/components/Login.css]: try to use classname for the buttons also. if possible.

File: src/components/Login.jsx [hunk 4]
```diff
@@ -31,14 +32,35 @@ function Login() {
...
+                <label>Password</label>
+            </div>
+            <div className="pass">Forgot Password?</div>
+            <button className = "button" onClick={login}>Login</button>
+            <div className="signup_link">
+                <p>Don't have an account? <Link to="/register">Register</Link></p>
+            </div>
+            </div>
+        </div>
+        /* <div class = "center">
```
Comment (sanjayguwaju) [inline src/components/Login.jsx]: write className as login-container

**Review by sanjayguwaju — CHANGES_REQUESTED** [review]
please change the code as requested.

File: src/components/Login.css (hunk 1 above)
Comment (smriteemanandhar) [reply src/components/Login.css]: Deleted

File: src/components/Login.css (hunk 2 above)
Comment (smriteemanandhar) [reply src/components/Login.css]: Deleted

File: src/components/Login.css (hunk 3 above)
Comment (smriteemanandhar) [reply src/components/Login.css]: Changed to className, buttonLogin

File: src/components/Login.jsx (hunk 4 above)
Comment (smriteemanandhar) [reply src/components/Login.jsx]: Done

**Review by sanjayguwaju — APPROVED** [review]

**Pull request merged**